tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -19,6 +23,22 @@ public String name() {
     return "places";
   }
 
+  // Evaluates place layer sort ordering of inputs into an integer for the sort-key field.
+  /*
+  static int getSortKey(float min_zoom, int population_rank, long population, String name) {
```

Comment:
@bdon Newbie Java guy here, super confused by all the type conversions and related errors, so this is pseudocode "working" but various compiler errors about type conversions...

Reply:
Giving this a try, thanks!